Description: Fix DetermintantalRepresentations package test on i386
 Use more forgiving epsilons for the clean function to avoid failures.
Author: Doug Torrance <dtorrance@piedmont.edu>
Bug: https://github.com/Macaulay2/M2/issues/1561
Forwarded: https://github.com/Macaulay2/M2/pull/1562
Last-Update: 2020-10-27

--- a/M2/Macaulay2/packages/DeterminantalRepresentations.m2
+++ b/M2/Macaulay2/packages/DeterminantalRepresentations.m2
@@ -1349,9 +1349,9 @@
 SCC=CC[x1,x2,x3]
 fCC=sub(f, SCC)
 reps = detRep fCC
-assert(all(reps, L -> clean(1e-10, fCC - det L) == 0))
+assert(all(reps, L -> clean(1e-9, fCC - det L) == 0))
 reps = detRep(fCC, HyperbolicPt => matrix{{3_RR},{2},{-7}}) -- representations are not monic
-assert(all(reps, L -> clean(1e-8, fCC - det L) == 0))
+assert(all(reps, L -> clean(1e-6, fCC - det L) == 0))
 ///
 
 TEST /// -- Cubic case: homogeneous, 3 variables
